src/index.js

```diff
@@ -765,7 +765,7 @@ if (nativeBindings.nativeMl) {
     if (layer) {
       const canvas = layer.source;
 
-      if (!mlPresentState.mlGlContext) {
+      if (mlPresentState.glContext !== canvas._context) {
```

Comment:
I think it's called `mlGlContext` in the code.

Reply:
Yeah sorry. Just renamed. Maybe good to eventually rename all of them? Anything special with mlGLContext?

Reply:
Nothing special, it's a legacy artifact of the code. It just won't function unless the names match.